src/appleseed/renderer/modeling/bssrdf/directionaldipolebssrdf.cpp

```diff
@@ -184,19 +186,6 @@ namespace
                 outgoing_point.get_point(),
                 outgoing_point.get_shading_normal(),
                 value);
-
```

Comment:
Keep this. We'll want it eventually, and it doesn't hurt as long as it's commented out.